Gruntfile.js

```diff
@@ -234,6 +234,10 @@ module.exports = function(grunt) {
   // Default task.
   grunt.registerTask('default', ['browserify']);
 
+  grunt.registerTask('set-test-env', function() {
+    process.env.NODE_ENV = 'test';
+  });
```

Comment:
This will not work when running the tests manually via `mocha`. I am proposing to use [env-test](https://www.npmjs.com/package/env-test) module instead and configure the environment in `test/mocha.opts`.